sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/core/EclipsePlugin.scala

```diff
@@ -225,7 +225,7 @@ trait EclipsePlugin {
 
   object EclipseProjectFlavor extends Enumeration {
 
-    val Scala = Value
+    val ScalaIDE = Value
```

Comment:
@jsuereth is the version number here for the version that's about to be release? i.e. 4.0.0?